### DISCUSSION TITLE
CoreNLP model on Spanish

### DISCUSSION THREAD

**Discussion by CoSineZxc** [stanfordnlp/CoreNLP#1274] [Q&A]
Is it possible to do constituent parsing based on models from CoreNLP?

**Comment by AngledLuffa** [comment]
Yes, the CoreNLP client in stanza should already handle that:

https://stanfordnlp.github.io/stanza/corenlp_client.html

One thing to note is that the trees do not get converted to the same ParseTree object that the stanza constituency processor adds
